### PR TITLE
pro-tools folder added and fix some other questions

### PR DESCRIPTION
**Pull request by emanuele-em**

### CODE REVIEW

File: adobe-photoshop/adobe-photoshop-quiz.md [hunk 1]
```diff
@@ -552,6 +552,7 @@
 
 -[x] anti-aliasing
 
+<<<<<<< HEAD
```
Comment (Ebazhanov) [inline adobe-photoshop/adobe-photoshop-quiz.md:555]:
```suggestion
<<<<<<< HEAD
```

i think you have to double check there were merge conflict

File: adobe-photoshop/adobe-photoshop-quiz.md [hunk 2]
```diff
@@ -565,3 +566,53 @@
...
+- [ ] The spot healing brush tool does not match texture, lifghting, shading or trasparency of the sampled pixels, but the healing brush tool blends texture, light, shading and trasparency of pixel nearby.
+- [ ] The healing brush tool does not match texture, lighting, shading or trasparency of the sampled pixels, but the spot healing brush tool blends texture, light, shading and trasparency of pixel nearby.
+
+#### Q80. Which tool is _not_ found in the *Select and Mask* workspace?
+
+- [ ] Quick select
+- [ ] Magic wand
+- [ ] Lasso
+- [x] Refine Edge
+>>>>>>> f74acb28a75e76473463845a4dd4da7316f8f5fb
```
Comment (Ebazhanov) [inline adobe-photoshop/adobe-photoshop-quiz.md:618]: and here

File: adobe-photoshop/adobe-photoshop-quiz.md [hunk 3]
```diff
@@ -480,7 +480,7 @@
 
 - [ ] It converts the image from a raster image to a vector infographics
 - [ ] It uses Adobe Sensei technology to reconstruct pixels that have been interpolated.
-- [ ] It references an image in a separate file that retains its resolutions.
+- [x] It references an image in a separate file that retains its resolutions.
```
Comment (emanuele-em) [inline adobe-photoshop/adobe-photoshop-quiz.md:483]: ok, it's correct

File: adobe-photoshop/adobe-photoshop-quiz.md (hunk 1 above)
Comment (emanuele-em) [reply adobe-photoshop/adobe-photoshop-quiz.md:555]: I don't know why I forgot this

File: adobe-photoshop/adobe-photoshop-quiz.md (hunk 2 above)
Comment (emanuele-em) [reply adobe-photoshop/adobe-photoshop-quiz.md:618]: I added Q74 at the end because there was a conflict with my local copy,

**Review by emanuele-em — COMMENTED** [review]
I forget to pull request for every commit and so I made some mistakes with conflicts but now I hope I explained everything

**Comment by Ebazhanov** [comment]
thanks @emanuele-em I will check tomorrow with fresh eyes

**Comment by Ebazhanov** [comment]
> thanks @emanuele-em I will check tomorrow with fresh eyes

I still believe you can fix merge commit

**Comment by emanuele-em** [comment]
Sorry for late

**Comment by Ebazhanov** [comment]
> Sorry for late

all good 👍

**Review by Ebazhanov — APPROVED** [review]
Hello, @emanuele-em! 

I see you added new 2 new quizzes 🎉 , thank you I will take care about adding reference links to Readme file.

Have a great day and thank you for your valuable contribution! 💐

**Pull request merged**